### PR TITLE
Init reform-scan on demo

### PR DESCRIPTION
**Pull request by rkondratowicz**

### CODE REVIEW

**Comment by doncem** [comment]
weren't these files generated with some script?

File: k8s/demo/common/reform-scan/identity.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: reform-scan
+  namespace: reform-scan
+spec:
+  type: 0
+  ResourceID: "/subscriptions/d025fece-ce99-4df2-b7a9-b649d3ff2060/resourcegroups/managed-identities-aat-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/reform-scan-demo-mi"
```
Comment (nitinprabhu) [inline k8s/demo/common/reform-scan/identity.yaml:8]: aat-rg ?

Comment (rkondratowicz) [reply k8s/demo/common/reform-scan/identity.yaml:8]: oops

File: k8s/demo/common/reform-scan/tests-service-account.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tests-service-account
+  namespace: reform-scan
```
Comment (mdayican) [inline k8s/demo/common/reform-scan/tests-service-account.yaml:5]: why do we need this file?

File: k8s/demo/common/reform-scan/identity.yaml (hunk 1 above)
Comment (rkondratowicz) [reply k8s/demo/common/reform-scan/identity.yaml]: fixed

**Pull request merged**